spec/requests/api/v1/menu_spec.rb

```diff
@@ -19,134 +18,132 @@
+                                        'ingredients' => @menu_items[2].ingredients
+                                      }
+                                    ]
+                                  }])
     end
 
     it 'returns a menu by :id' do
       menu = @menus[0]
       get "/v1/menus/#{menu.id}"
-      expect(response_json).to eq({'menu' =>
-                                       {'id' => @menus[0].id,
-                                        'title' => @menus[0].title,
-                                        'start_date' => @menus[0].start_date.strftime('%F'),
-                                        'end_date' => @menus[0].end_date.strftime('%F')},
-                                   'uri' => "http://www.example.com/v1/menus/#{@menus[0].id}",
-                                   'item_count' => 3,
-                                   'items' =>
-                                       [
-                                           {
-                                               'item' => @menu_items[0].name,
-                                               'price' => @menu_items[0].price.to_f.to_s,
-                                               'description' => @menu_items[0].description,
-                                               'ingredients' => @menu_items[0].ingredients
-                                           },
-                                           {
-                                               'item' => @menu_items[1].name,
-                                               'price' => @menu_items[1].price.to_f.to_s,
-                                               'description' => @menu_items[1].description,
-                                               'ingredients' => @menu_items[1].ingredients
-                                           },
-                                           {
-                                               'item' => @menu_items[2].name,
-                                               'price' => @menu_items[2].price.to_f.to_s,
-                                               'description' => @menu_items[2].description,
-                                               'ingredients' => @menu_items[2].ingredients
-                                           }
-                                       ]
-                                  })
+      expect(response_json).to eq('id' => @menus[0].id,
+                                  'title' => @menus[0].title,
+                                  'start_date' => @menus[0].start_date.strftime('%F'),
+                                  'end_date' => @menus[0].end_date.strftime('%F'),
+                                  'uri' => "http://www.example.com/v1/menus/#{@menus[0].id}",
+                                  'item_count' => 3,
+                                  'items' => [
+                                    {
+                                      'id' => @menu_items[0].id,
+                                      'name' => @menu_items[2].name,
```

Comment:
`@menu_items[2].name` instead of `@menu_items[0].name` and the tests are passing? Something is wrong.